### PR TITLE
Add python-chado and python-tripal packages

### PR DESCRIPTION
**Pull request by abretaud**
* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/chado/bld.bat [hunk 1]
```diff
@@ -0,0 +1,2 @@
+"%PYTHON%" setup.py install
```
Comment (abretaud) [inline recipes/chado/bld.bat]: Do you mean removing the .bat?

Comment (bgruening) [inline recipes/chado/bld.bat]: yes

**Comment by abretaud** [comment]
@bgruening a little +1 to merge this?

**Comment by bgruening** [comment]
:+1:

**Comment by abretaud** [comment]
jolly good, thanks!

**Pull request merged**